test/lib/usd/translators/testUsdImportMaterialX.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env mayapy
+#
+# Copyright 2021 Autodesk
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from maya import cmds
+from maya import standalone
+
+import os
+import unittest
+
+import fixturesUtils
+
+try:
+    from pxr import UsdMtlx
+    USD_HAS_MATERIALX_PLUGIN=True
+except ImportError:
+    USD_HAS_MATERIALX_PLUGIN=False
+
+
+class testUsdImportMaterialX(unittest.TestCase):
+    """Test importing an asset with MaterialX shading:
```

Comment:
Excellent test, @JGamache-autodesk! The only thing that would be good to add (but it's not blocking for this merge) is a test that performs export and import. I know part of the import file is actually having content exported from Maya, but I would want to avoid situations where USD test file gets updated but not the actual export code.